iFixit/react-commerce
# Use new back in stock api

closes #688 

This PR updates the notify me box (for back in stock notifications) to use the new api introduced with https://github.com/iFixit/ifixit/pull/44619

## QA

1. Visit the Vercel preview
2. Go to an [out of stock product page](https://react-commerce-git-use-new-back-in-stock-api-ifixit.vercel.app/Products/17-blade-5-pcs)
3. Using the notify me box try to subscribe, and verify that the new `notifyWhenSkuInStock` endpoint is called
